Sources/xcodeproj/Objects/BuildPhase/PBXBuildFile.swift

```diff
@@ -94,16 +100,10 @@ extension PBXBuildFile {
     /// - Returns: build phase name.
     /// - Throws: an error if the name cannot be obtained.
     func buildPhaseName() throws -> String? {
-        let type = try buildPhaseType()
-        let projectObjects = try objects()
-        switch type {
-        case .copyFiles?:
-            return projectObjects.copyFilesBuildPhases
```

Comment:
This also removes the double loop of copy build phase files, even if there wasn't a cache